documentation/book/con-product-downloads.adoc

```diff
@@ -5,14 +5,12 @@
 [id='downloads-{context}']
 = {ProductName} Downloads
 
-{ProductName} releases are available for download from {ReleaseDownload}.
 ifdef::Downloading[]
-The release artifacts contain documentation
-and example YAML files for deployment on {ProductPlatformName}.
-The example files are used throughout this documentation and can be used to install {ProductName}.
-There is a Helm Chart for deployment the Cluster Operator, using link:https://helm.sh/[Helm^], as well.
+{ProductName} releases are available to download from {ReleaseDownload}. The release artefacts contain documentation and example `.yaml` files for deployment on {ProductPlatformName}. The example files are used throughout this documentation and can be used to install {ProductName}. Additionally, a Helm Chart is provided for deploying the Cluster Operator using link:https://helm.sh/[Helm^]. The container images are available through the {DockerRepository}.
```

Comment:
Writing the documents with only one sentence per line would make it easier to comment on the exact sentence.

Reply:
It would, but we risk running into problems if we ever convert the documentation from AsciiDoc to another format. But you're right, it is difficult to pinpoint comments on a particular sentence. I'll have a think about how we could improve this.

Reply:
Well ... 99% of the documentation is done that way ... so I'm not sure whether there is really anything to worry about :-).